CMakeLists.txt

```diff
@@ -1,6 +1,35 @@
-cmake_minimum_required(VERSION 3.19 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.21 FATAL_ERROR)
 
 cmake_policy(SET CMP0091 NEW)
```

Comment:
We should add a comment explaining what these are (something like `SET CMP00091 NEW) # Select MSVC runtime library flags using an abstraction`) just for future reference.